integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -64,6 +64,15 @@ spec:
       {{- include "alluxio.hostAliases" . | nindent 6 }}
       {{ end }}
       hostNetwork: {{ $hostNetwork }}
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
I prefer the "softer" version `preferredDuringSchedulingIgnoredDuringExecution`. I know that some users do not mind running multiple master pods on the same node (for example, 2 nodes for 3 master pods). This deployment makes sense for test and dev environments. 

Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity

Reply:
@jiacheliu3 
I think I can move `affinity` to values.yaml, so that users  can decide whether to use k8s's affinity. Many helm charts come with this optional configuration.

Reply:
Sure